assets/src/scss/layout/_footer.scss

```diff
@@ -1,240 +1,239 @@
 .site-footer {
   --footer-- {
     font-family: $roboto;
-    background: $dark-blue;
+    background-color: $dark-blue;
   }
-  padding: 45px 0 $n25;
+
   position: relative;
   z-index: 2;
   text-align: center;
   transition: margin 1s;
+  padding-top: $n25;
   clear: both;
-  line-height: 0;
-
-  @include medium-and-up {
-    padding: $n25 0 $n20;
-  }
-
-  @include large-and-up {
-    padding: 55px 0 $n25;
-  }
 
   a --footer--links-- {
-    color: inherit;
+    color: $white;
 
-    &:hover {
-      color: $white;
-    }
+    // &:hover {
```

Comment:
This is here intentionally: `--footer--links--` in the selector turns all properties into variables. So where in the default theme they have the same color, it allows a theme to have a different hover color.

Reply:
However this is also a weird case: there's other color variables for different links in the footer. So probably it's better to remove this one and ensure each part can be set individually.